Refactor format command fromversion key (#1934)
demisto/demisto-sdk: CHANGELOG.md

* Fixed a bug where **doc-review** command failed on existing templates.
* Fixed a bug where **validate** command failed when the word demisto is in the repo README file.
* Added support for adding test-playbooks to the zip file result in *create-content-artifacts* command for marketplacev2.

> * Refactored the **format** command on update *fromversion*:

Reviewer: ```suggestion
* Refactored the **format** command's effect on the *fromversion* field:
```

>   - Fixed a bug when changing content item, the *fromversion* field should remain.

Reviewer: ```suggestion
  - Fixed a bug where the *fromversion* field was removed when modifying a content item.
```

>   - Updated the general default *fromversion* and the default *fromversion* of new content items (e.g. `Lists, Jobs..`).

Reviewer: ```suggestion
  - Updated the general default *fromversion* and the default *fromversion* of newly-introduced content items (e.g. `Lists`, `Jobs`).
```

>   - Removed the interactive in *update_playbook*.

Reviewer: ```suggestion
  - Removed the interactive option in *update_playbook*.
```

Reviewer: why is it removed, by the way?

Author: The logic is compromised when receiving input from the user at this current stage of updating the fromVersion field. In addition, an option of interactivity has been added to all content types as can be seen in the following line.

>   - Removed the interactive in *update_playbook*.

Reviewer: why?

>   - Added interactive for all content types, asking user if to generate default *fromversion*.

Reviewer: ```suggestion
  - Added an *interactive* mode for all content types, asking user if to set the default *fromversion*.
```

>   - Updated *update_generic_yml/json* and their usages in the differences content types update.

Reviewer: ```suggestion
  - Updated *update_generic_yml/json* and their behavior when content items are modified.
```

unclear, if my suggestion makes sense please commit it, otherwise please elaborate a bit more

Author: You are right, an unnecessary remark.
